### DISCUSSION TITLE
Aspect ratio lock for tablet driver works incorrectly

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#15785] [Q&A] [answered]
For example, I play only with the aspect ratio 1.44, but the game does not allow me to put this (sooo annoying):

https://user-images.githubusercontent.com/86934170/143288819-93e8ff35-3444-437c-b713-b410da5d607e.mp4

And even when I was finally able to put this ratio to width that im playing with (50) through a different value, it still differs by 0.5 mm from what is in OpenTabletDriver.
In the case of width 52, it generally differs by 1 mm.

![ScreenShot_20211124203955](https://user-images.githubusercontent.com/86934170/143289060-85dd396e-0a3f-4c2d-93d1-b380f43a91b0.png)
![ScreenShot_20211124204019](https://user-images.githubusercontent.com/86934170/143289063-8d74d82c-cec9-459e-bc16-c5143ef1e2e4.png)

To solve this, I think so, is to increase the precision through https://github.com/ppy/osu/discussions/13188

Please don't tell me use manual configure for 52x36 to turn in 1.44. This is not a salvation. I just want to set ratio and configure my width, maybe, for little more or less of 52.

**Comment by peppy** [comment, marked as answer]
please post in the existing discussion you rightfully linked rather than opening a new one.

**Reply by Loreos7** [reply]
There is no discussion with my problem.
Ok, just replied with the link to this.
